pkg/auth/providers/common/password.go

```diff
@@ -0,0 +1,61 @@
+package common
+
+import (
+	"fmt"
+	"reflect"
+	"strings"
+
+	corev1 "github.com/rancher/types/apis/core/v1"
+	"k8s.io/api/core/v1"
+	apierrors "k8s.io/apimachinery/pkg/api/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+const Namespace = "mgmt-secrets"
```

Comment:
lets rename the constant so it reflects the fact that it's created for secrets only